### PR TITLE
question page update #41

### PR DESCRIPTION
**Pull request by Sanzouille**
added infos on registering, name changing, notifications, client

### CODE REVIEW

**Comment by Sanzouille** [comment]
What tag are you talking about?

File: www/questions/index.php [hunk 1]
```diff
@@ -31,10 +31,25 @@
     <p>Be fair.</p>
     <h2>What are your plans?</h2>
     <p>Expand. We want your participation.</p>
+    <h2>What is the MatchClient?</h2>
+    <p>The MatchClient is a custom client we made to better suits the needs of clanwars. 
+    With it you will be able to access compatible servers that support features like /pause !</p>
```
Comment (ScalaWilliam) [inline www/questions/index.php]: put /pause in bold too

File: www/questions/index.php [hunk 2]
```diff
@@ -31,10 +31,25 @@
...
+    <h2>Why should I register?</h2>
+    <p>When registering, it creates your profile page where you can find statistics about all your games on our servers. 
+    You will also unlock achievements while playing!</p>
     <h2>Why Google sign in?</h2>
     <p>Saves us time in development.</p>
-    <h2>Why is the homepage slow?</h2>
-    <p>It's fast on Google Chrome. We are using Polymer/Web Components which save us time.</p>
+    <h2>Can I change the in-game name i picked while registering?</h2>
+    <p>In the current state you are not able to change it. 
+        However if the name change is necessary (i.e joined or left a clan..) then you can contact Sanzo that will do it for you. </p>
```
Comment (ScalaWilliam) [inline www/questions/index.php]: contact you how?

**Comment by ScalaWilliam** [comment]
:+1:

**Pull request merged**